libraries/chain/include/cyberway/genesis/genesis_ee_container.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <eosio/chain/types.hpp>
+#include <fc/reflect/reflect.hpp>
+
+namespace cyberway { namespace genesis {
+
+using namespace eosio::chain;
+
+struct genesis_ee_header {
+    char magic[14] = "CyberwayEEGen";
+    uint32_t version = 1;
+
+    bool is_valid() {
+        genesis_ee_header oth;
+        return string(magic) == oth.magic && version == oth.version;
+    }
+};
+
+enum class table_ee_type {
+    messages
+};
+
+struct table_ee_header {
+    table_ee_type type;
+
+    table_ee_header(table_ee_type type_)
+            : type(type_) {
+    }
+};
+
+struct message_ee_object {
+    account_name parent_author;
+    string parent_permlink;
+    account_name author;
+    string permlink;
+    string title;
+    string body;
+    string json_metadata;
```

Comment:
json_metadata ? 
we need only lang and tags from it.

Reply:
@afalaleev Removed it, added tags.